ground/gcs/src/plugins/uavsettingsimportexport/uavsettingsimportexportfactory.cpp

```diff
@@ -404,7 +265,6 @@ QString UAVSettingsImportExportFactory::createXMLDocument(const enum storedData
     // create settings and/or data elements
     QDomElement settings = doc.createElement("settings");
     QDomElement data = doc.createElement("data");
-    QDomElement waypoints = doc.createElement("waypoints");
```

Comment:
That's correct, we had three, all slightly overlapping but not entirely consistent. I don't see the point of including a fixed flight plan in a UAV Config, this sounds very dangerous.